samples/net/ocpp/src/main.c

```diff
@@ -0,0 +1,384 @@
+/*
+ * Copyright (c) 2024 Linumiz
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <stdio.h>
+
+#include <zephyr/net/net_if.h>
+#include <zephyr/net/net_core.h>
+#include <zephyr/net/net_context.h>
+#include <zephyr/net/net_mgmt.h>
+#include <zephyr/net/sntp.h>
+#include <zephyr/net/ocpp.h>
+#include <zephyr/posix/time.h>
+#include <zephyr/random/random.h>
+#include <zephyr/zbus/zbus.h>
+
+LOG_MODULE_REGISTER(main, LOG_LEVEL_INF);
+
+#define NO_OF_CONN 2
+K_KERNEL_STACK_ARRAY_DEFINE(cp_stk, NO_OF_CONN, 2 * 1024);
+struct k_thread tinfo[NO_OF_CONN];
+k_tid_t tid[NO_OF_CONN];
+char idtag[NO_OF_CONN][25];
+
+#if defined(CONFIG_NET_DHCPV4)
+static struct net_mgmt_event_callback ocpp_dhcp_cb;
+
+static void ocpp_handler_cb(struct net_mgmt_event_callback *cb,
+			    uint32_t mgmt_event, struct net_if *iface)
+{
+	char buf[NET_IPV4_ADDR_LEN];
+
+	LOG_INF("net mgr cb");
+	if (mgmt_event != NET_EVENT_IPV4_DHCP_BOUND) {
+		return;
+	}
+
+	LOG_INF("Your address: %s",
+			net_addr_ntop(AF_INET,
+				&iface->config.dhcpv4.requested_ip,
+				buf, sizeof(buf)));
+	LOG_INF("Lease time: %u seconds",
+			iface->config.dhcpv4.lease_time);
+	LOG_INF("Subnet: %s",
+			net_addr_ntop(AF_INET,
+				&iface->config.ip.ipv4->netmask,
+				buf, sizeof(buf)));
+	LOG_INF("Router: %s",
+			net_addr_ntop(AF_INET,
+				&iface->config.ip.ipv4->gw,
+				buf, sizeof(buf)));
+}
+
+static int ocpp_test_dhcp_init(void)
+{
+	struct net_if *iface;
+
+	net_mgmt_init_event_callback(&ocpp_dhcp_cb, ocpp_handler_cb,
+			NET_EVENT_IPV4_DHCP_BOUND);
```

Comment:
here too, the `NET_EVENT...` should start from the same column as `&ocpp` in previous line.